gen_cert.py

```diff
@@ -85,14 +85,18 @@ def get_cert_date(
     - If using the "ROLLING" certs feature, use today's date
     - If all else fails use 'configured_date_parameter' for date
     """
-
+    utc_zone = get_timezone('UTC')
+    to_zone = get_timezone(timezone)
+    
     if calling_date_parameter:
-        date_value = format_datetime(calling_date_parameter, 'MMMM d, y', tzinfo=timezone, locale=locale)
+        date = strptime(calling_date_parameter, '%y-%m-%d')
     elif configured_date_parameter == "ROLLING":
-        date_value = format_datetime(datetime.datetime.today(), 'MMMM d, y', tzinfo=timezone, locale=locale)
+        date = datetime.datetime.today()
```

Comment:
From the docs [1]:

> classmethod datetime.now([tz])¶
>    Return the current local date and time. If optional argument tz is None or not specified, this is like today(), but, if possible, supplies more precision than can be gotten from going through a time.time() timestamp (for example, this may be possible on platforms supplying the C gettimeofday() function).

>    If tz is not None, it must be an instance of a tzinfo subclass, and the current date and time are converted to tz’s time zone. In this case the result is equivalent to tz.fromutc(datetime.utcnow().replace(tzinfo=tz)). See also today(), utcnow().

So it looks like we could actually use `today`, pass in the timezone=utc, and avoid the call to `date.replace`.

[1] https://docs.python.org/2/library/datetime.html#datetime.datetime.now